<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Seareiros</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../res/icons.qrc">
    <normaloff>:/icons/title.png</normaloff>:/icons/title.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Arquivo</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRegisters">
    <property name="title">
     <string>&amp;Cadastros</string>
    </property>
    <addaction name="actionAddAssociate"/>
    <addaction name="actionAddActivity"/>
    <addaction name="actionAddBook"/>
    <addaction name="actionAddProduct"/>
   </widget>
   <widget class="QMenu" name="menuSales">
    <property name="title">
     <string>&amp;Vendas</string>
    </property>
    <addaction name="actionSellBook"/>
    <addaction name="actionSellProduct"/>
    <addaction name="actionSellEvent"/>
   </widget>
   <widget class="QMenu" name="menuReports">
    <property name="title">
     <string>&amp;Relatórios</string>
    </property>
    <addaction name="actionRepSales"/>
    <addaction name="actionRepBook"/>
    <addaction name="actionRepProduct"/>
    <addaction name="actionRepLibrary"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuLibrary">
    <property name="title">
     <string>&amp;Biblioteca</string>
    </property>
    <addaction name="actionLibLend"/>
    <addaction name="actionLibReturn"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Editar</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuRegisters"/>
   <addaction name="menuSales"/>
   <addaction name="menuLibrary"/>
   <addaction name="menuReports"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSellBook"/>
   <addaction name="actionSellProduct"/>
   <addaction name="actionSellEvent"/>
   <addaction name="separator"/>
   <addaction name="actionSearch"/>
   <addaction name="actionPendencies"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../../../res/icons.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../../res/icons.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>Sobre</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAddAssociate">
   <property name="icon">
    <iconset resource="../../../res/icons.qrc">
     <normaloff>:/icons/associate_add.png</normaloff>:/icons/associate_add.png</iconset>
   </property>
   <property name="text">
    <string>Associado</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSellBook">
   <property name="icon">
    <iconset resource="../../../res/icons.qrc">
     <normaloff>:/icons/book.png</normaloff>:/icons/book.png</iconset>
   </property>
   <property name="text">
    <string>Livros</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSellProduct">
   <property name="icon">
    <iconset resource="../../../res/icons.qrc">
     <normaloff>:/icons/product.png</normaloff>:/icons/product.png</iconset>
   </property>
   <property name="text">
    <string>Bazar</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSellEvent">
   <property name="icon">
    <iconset resource="../../../res/icons.qrc">
     <normaloff>:/icons/events.png</normaloff>:/icons/events.png</iconset>
   </property>
   <property name="text">
    <string>Eventos</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAddBook">
   <property name="icon">
    <iconset resource="../../../res/icons.qrc">
     <normaloff>:/icons/book_add.png</normaloff>:/icons/book_add.png</iconset>
   </property>
   <property name="text">
    <string>Livro</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAddProduct">
   <property name="icon">
    <iconset resource="../../../res/icons.qrc">
     <normaloff>:/icons/product_add.png</normaloff>:/icons/product_add.png</iconset>
   </property>
   <property name="text">
    <string>Produto</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionLibLend">
   <property name="icon">
    <iconset resource="../../../res/icons.qrc">
     <normaloff>:/icons/book_lend.png</normaloff>:/icons/book_lend.png</iconset>
   </property>
   <property name="text">
    <string>Alugar</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionLibReturn">
   <property name="icon">
    <iconset resource="../../../res/icons.qrc">
     <normaloff>:/icons/book_return.png</normaloff>:/icons/book_return.png</iconset>
   </property>
   <property name="text">
    <string>Devolver</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../../../res/icons.qrc">
     <normaloff>:/icons/preferences.png</normaloff>:/icons/preferences.png</iconset>
   </property>
   <property name="text">
    <string>Preferências</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../res/icons.qrc">
     <normaloff>:/icons/copy.png</normaloff>:/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copiar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRepSales">
   <property name="text">
    <string>Vendas</string>
   </property>
  </action>
  <action name="actionRepLibrary">
   <property name="text">
    <string>Biblioteca</string>
   </property>
  </action>
  <action name="actionRepProduct">
   <property name="text">
    <string>Bazar</string>
   </property>
  </action>
  <action name="actionRepBook">
   <property name="text">
    <string>Livros</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="../../../res/icons.qrc">
     <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
   </property>
   <property name="text">
    <string>Pesquisar</string>
   </property>
   <property name="toolTip">
    <string>Pesquisar</string>
   </property>
  </action>
  <action name="actionPendencies">
   <property name="icon">
    <iconset resource="../../../res/icons.qrc">
     <normaloff>:/icons/pending.png</normaloff>:/icons/pending.png</iconset>
   </property>
   <property name="text">
    <string>Pendências</string>
   </property>
   <property name="toolTip">
    <string>Pendências</string>
   </property>
  </action>
  <action name="actionAddActivity">
   <property name="icon">
    <iconset resource="../../../res/icons.qrc">
     <normaloff>:/icons/activity_add.png</normaloff>:/icons/activity_add.png</iconset>
   </property>
   <property name="text">
    <string>Atividade</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../res/icons.qrc"/>
 </resources>
 <connections/>
</ui>
